api/peers.go

```diff
@@ -88,9 +88,19 @@ func (h *PeerHandlers) ConnectionsHandler(w http.ResponseWriter, r *http.Request
 
 func (h *PeerHandlers) listPeersHandler(w http.ResponseWriter, r *http.Request) {
 	args := core.ListParamsFromRequest(r)
-	args.OrderBy = "created"
+	// args.OrderBy = "created"
+	cached, err := util.ReqParamBool("cached", r)
```

Comment:
If you just hit /peers with no "cached" param, you get this error: strconv.ParseBool: parsing "": invalid syntax

I don't think we need to care about the error? assuming that ReqParamBool will only have cached == true if we supply "true" or "t" anything else should be false anyway

Reply:
ah shit yeah I forgot about this. Will fix. At some point we should PR `datatogether/api/apiutil` to add a default param to `ReqParamBool` so that it returns the default value when no value is present, instead of erroring.